src/Test/Unit/Util/UrlManagerTest.php

```diff
@@ -461,12 +461,21 @@ public function testGetBaseUrlWithEmptyStoreUrls()
 
         $this->magentoShellMock->expects($this->once())
             ->method('execute')
-            ->with('config:show:store-url default')
+            ->with('config:show:default-url')
             ->willThrowException(new ShellException('some error'));
 
         $this->environmentMock->expects($this->once())
             ->method('getRoutes')
             ->willReturn(['http://example.com/' => ['original_url' => 'https://{default}', 'type' => 'upstream']]);
+        $this->loggerMock->expects($this->once())
+            ->method('error')
+            ->with('Cannot fetch base URL using the config:show:default-url command. URL from routes will be used');
```

Comment:
From MAGENTO-CLOUD-ROUTES defined with the routes.yaml file? Maybe should indicate where the routes are so they remember where to find? Just a thought...

```suggestion
            ->with('Cannot fetch base URL using the config:show:default-url command. Instead, using the URL from the .magento/routes.yaml file.');
```

Reply:
Actually, in such case URLs are taking from environment variable not from `.magento/routes.yaml`  file it can be confusing.

Reply:
Ah, okay, I'll update that suggestion.